src/main/java/seedu/edrecord/logic/commands/DeleteAssignmentCommand.java

```diff
@@ -24,10 +24,10 @@
 
     public static final String MESSAGE_DELETE_ASSIGNMENT_SUCCESS = "Deleted assignment: %1$s";
 
-    private final Index targetIndex;
+    private final Index index;
 
```

Comment:
Can change the usage string (line 21) from "index" to "ID" (I can't comment on unchanged code sry)

Reply:
Fixed in #225